src/app/_components/ScreenSizeIndicator.tsx

```diff
@@ -0,0 +1,14 @@
+export function ScreenSizeIndicator() {
```

Comment:
What about `ResponsiveDebugger`. I feel it something like that might indicate it's used for Development purposes.

I think we can also prefix this component with an `_` - What do you think?

Reply:
Yes, good points.

I renamed the component `BreakpointDebbuger` and changed its path to `_dev_BreakpointDebbuger.tsx`, so that it's clear that it's for dev only.

Let me know what you think :)